spec/grid_spec.rb

```diff
@@ -179,4 +157,17 @@ def game_with_winner
     expect(grid.initial_state?).to eq(false)
   end
 
+  it "returns copies of grid state each with mark placed on different empty cell" do
+    grid.place_mark("3", "X")
+    grid.place_mark("5", "O")
```

Comment:
I still think it would be better to make the Grid class flexible so you can instantiate it with an array of cells. So you could do something like `Grid.new(['X', 1,2,3,'O',5,6,'X'])` and don't need to call the `place_mark` method several times.

Reply:
Maybe have a think how you would do it and we can discuss it in the IPM tomorrow.